File: js/lizardfolkNames.js
var nm1 = ["","","","","bh","br","ch","dr","g","gr","j","k","kr","q","qr","r","rh","th","thr","tr","v","y","z"];
var nm2 = ["a","i","o","u","a","i","o","u","a","i","o","u","a","i","o","u","a","i","o","u","a","i","o","u","a","i","o","u","a","i","o","u","a","i","o","u","a","i","o","u","ao","au","ou","iu","oa","ua","uu"];
var nm3 = ["g","gr","h","j","k","kz","kx","kn","lt","ll","m","mz","nt","nj","r","rq","rs","rz","rn","rl","sh","sz","shr","szr","t","tt","th","tr","thr","tz","tsz","tl","x","xl","z","zz"];
var nm4 = ["c","cc","ch","g","j","k","kk","q","x","xx","z"];
var nm5 = ["","","","k","sz","sk","shk","t","x","xl","z","zk"];

var nm6 = ["","","","","","","b","cr","ch","d","dh","dr","h","j","k","kh","kr","q","r","rh","s","sh","sr","shr","th","thr","y","z"];
var nm7 = ["a","e","i","o","u","a","e","i","o","u","a","e","i","o","u","a","e","i","o","u","a","e","i","o","u","a","e","i","o","u","a","e","ae","ie","ia","ei"];
var nm8 = ["c","cc","ch","d","dr","g","gn","gz","gy","k","kk","kr","kz","kl","kn","lq","lz","n","nq","nr","nz","r","rs","rq","rt","rz","rsh","rzh","t","tt","th","thr","thl","tl","z","zd"];
var nm9 = ["c","j","k","q","r","s","t","x","y","z"];
var nm10 = ["","","","","h","s","sk","ss","x","z"];

var nm11 = ["","","","","","b","ch","d","dr","j","k","kh","kr","q","r","rh","sr","shr","th","thr","y","z"];
var nm12 = ["a","e","i","o","u","a","e","i","o","u","a","e","i","o","u","a","e","i","o","u","a","e","i","o","u","a","e","i","o","u","a","e","u","u","u","ae","ia","ao","au","ou","oa"];
var nm13 = ["g","gr","k","kk","kz","kl","kn","lz","r","rs","rq","rt","rz","t","tt","th","thr","tl","z","zd"];
var nm14 = ["c","j","k","q","r","t","x","z"];
var nm15 = ["","","","","s","sk","ss","x","xl","z"];

var br = "";

function nameGen(type){
	$('#placeholder').css('textTransform', 'capitalize');
	var tp = type;
	var element = document.createElement("div");
	element.setAttribute("id", "result");
	
	for(i = 0; i < 10; i++){
		rnd2 = Math.random() * nm3.length | 0;
		if(tp === 1){
			rnd = Math.random() * nm6.length | 0;
			rnd2 = Math.random() * nm7.length | 0;
			rnd3 = Math.random() * nm10.length | 0;
			if(i < 3){
				while(rnd < 6){
					rnd = Math.random() * nm6.length | 0;
				}
				while(rnd3 < 4){
					rnd3 = Math.random() * nm10.length | 0;
				}
				names = nm6[rnd] + nm7[rnd2] + nm10[rnd3];
			}else if(i < 6){
				rnd4 = Math.random() * nm7.length | 0;
				rnd5 = Math.random() * nm8.length | 0;
				while(nm10[rnd3] === nm8[rnd5]){
					rnd5 = Math.random() * nm8.length | 0;
				}
				names = nm6[rnd] + nm7[rnd2] + nm8[rnd5] + nm7[rnd4] + nm10[rnd3];
			}else{
				rnd4 = Math.random() * nm7.length | 0;
				rnd5 = Math.random() * nm8.length | 0;
				rnd6 = Math.random() * nm7.length | 0;
				rnd7 = Math.random() * nm9.length | 0;
				while(nm10[rnd3] === nm8[rnd5]){
					rnd5 = Math.random() * nm8.length | 0;
				}
				while(nm9[rnd7] === nm8[rnd5]){
					rnd7 = Math.random() * nm9.length | 0;
				}
				if(i < 8){
					names = nm6[rnd7] + nm7[rnd2] + nm9[rnd7] + nm7[rnd4] + nm8[rnd5] + nm7[rnd6] + nm10[rnd3];
				}else{
					names = nm6[rnd7] + nm7[rnd4] + nm8[rnd5] + nm7[rnd2] + nm9[rnd7] + nm7[rnd6] + nm10[rnd3];
				}
			}
		}else if(tp === 2){
			rnd = Math.random() * nm11.length | 0;
			rnd2 = Math.random() * nm12.length | 0;
			rnd3 = Math.random() * nm15.length | 0;
			if(i < 3){
				while(rnd < 4){
					rnd = Math.random() * nm11.length | 0;
				}
				while(rnd3 < 4){
					rnd3 = Math.random() * nm15.length | 0;
				}
				names = nm11[rnd] + nm12[rnd2] + nm15[rnd3];
			}else if(i < 6){
				rnd4 = Math.random() * nm12.length | 0;
				rnd5 = Math.random() * nm13.length | 0;
				while(nm12[rnd3] === nm13[rnd5]){
					rnd5 = Math.random() * nm13.length | 0;
				}
				names = nm11[rnd] + nm12[rnd2] + nm13[rnd5] + nm12[rnd4] + nm15[rnd3];
			}else{
				rnd4 = Math.random() * nm12.length | 0;
				rnd5 = Math.random() * nm13.length | 0;
				rnd6 = Math.random() * nm12.length | 0;
				rnd7 = Math.random() * nm14.length | 0;
				while(nm15[rnd3] === nm13[rnd5]){
					rnd5 = Math.random() * nm13.length | 0;
				}
				while(nm14[rnd7] === nm13[rnd5]){
					rnd7 = Math.random() * nm14.length | 0;
				}
				if(i < 8){
					names = nm11[rnd7] + nm12[rnd2] + nm14[rnd7] + nm12[rnd4] + nm13[rnd5] + nm12[rnd6] + nm15[rnd3];
				}else{
					names = nm11[rnd7] + nm12[rnd4] + nm13[rnd5] + nm12[rnd2] + nm14[rnd7] + nm12[rnd6] + nm15[rnd3];
				}
			}
		}else{
			rnd = Math.random() * nm1.length | 0;
			rnd2 = Math.random() * nm2.length | 0;
			rnd3 = Math.random() * nm5.length | 0;
			if(i < 3){
				while(rnd < 4){
					rnd = Math.random() * nm1.length | 0;
				}
				while(rnd3 < 3){
					rnd3 = Math.random() * nm5.length | 0;
				}
				names = nm1[rnd] + nm2[rnd2] + nm5[rnd3];
			}else if(i < 6){
				rnd4 = Math.random() * nm2.length | 0;
				rnd5 = Math.random() * nm3.length | 0;
				while(nm2[rnd3] === nm3[rnd5]){
					rnd5 = Math.random() * nm3.length | 0;
				}
				names = nm1[rnd] + nm2[rnd2] + nm3[rnd5] + nm2[rnd4] + nm5[rnd3];
			}else{
				rnd4 = Math.random() * nm2.length | 0;
				rnd5 = Math.random() * nm3.length | 0;
				rnd6 = Math.random() * nm2.length | 0;
				rnd7 = Math.random() * nm4.length | 0;
				while(nm5[rnd3] === nm3[rnd5]){
					rnd5 = Math.random() * nm3.length | 0;
				}
				while(nm4[rnd7] === nm3[rnd5]){
					rnd7 = Math.random() * nm4.length | 0;
				}
				if(i < 8){
					names = nm1[rnd7] + nm2[rnd2] + nm4[rnd7] + nm2[rnd4] + nm3[rnd5] + nm2[rnd6] + nm5[rnd3];
				}else{
					names = nm1[rnd7] + nm2[rnd4] + nm3[rnd5] + nm2[rnd2] + nm4[rnd7] + nm2[rnd6] + nm5[rnd3];
				}
			}
		}
		br = document.createElement('br');	
		element.appendChild(document.createTextNode(names));
		element.appendChild(br);
	}
	if(document.getElementById("result")){
		document.getElementById("placeholder").removeChild(document.getElementById("result"));
	}		
	document.getElementById("placeholder").appendChild(element);
}

File: generators/lizardfolkNames.py
__all__ = ['lizardfolkNames']

# Don't look below, you will not understand this Python code :) I don't.

from js2py.pyjs import *
# setting scope
var = Scope( JS_BUILTINS )
set_global_object(var)

# Code follows:
var.registers(['nm1', 'nm11', 'nm12', 'nm4', 'nameGen', 'nm5', 'nm8', 'nm14', 'nm3', 'nm13', 'nm7', 'nm9', 'nm10', 'nm2', 'nm15', 'nm6'])
@Js
def PyJsHoisted_nameGen_(type, this, arguments, var=var):
    var = Scope({'arguments':arguments, 'this':this, 'type':type}, var)
    var.registers(['tp', 'result', 'type'])
    var.put('tp', var.get('type'))
    var.put('result', Js([]))
    #for JS loop
    var.put('i', Js(0.0))
    while (var.get('i')<Js(10.0)):
        try:
            var.put('rnd2', ((var.get('Math').callprop('random')*var.get('nm3').get('length'))|Js(0.0)))
            if PyJsStrictEq(var.get('tp'),Js(1.0)):
                var.put('rnd', ((var.get('Math').callprop('random')*var.get('nm6').get('length'))|Js(0.0)))
                var.put('rnd2', ((var.get('Math').callprop('random')*var.get('nm7').get('length'))|Js(0.0)))
                var.put('rnd3', ((var.get('Math').callprop('random')*var.get('nm10').get('length'))|Js(0.0)))
                if (var.get('i')<Js(3.0)):
                    while (var.get('rnd')<Js(6.0)):
                        var.put('rnd', ((var.get('Math').callprop('random')*var.get('nm6').get('length'))|Js(0.0)))
                    while (var.get('rnd3')<Js(4.0)):
                        var.put('rnd3', ((var.get('Math').callprop('random')*var.get('nm10').get('length'))|Js(0.0)))
                    var.put('names', ((var.get('nm6').get(var.get('rnd'))+var.get('nm7').get(var.get('rnd2')))+var.get('nm10').get(var.get('rnd3'))))
                else:
                    if (var.get('i')<Js(6.0)):
                        var.put('rnd4', ((var.get('Math').callprop('random')*var.get('nm7').get('length'))|Js(0.0)))
                        var.put('rnd5', ((var.get('Math').callprop('random')*var.get('nm8').get('length'))|Js(0.0)))
                        while PyJsStrictEq(var.get('nm10').get(var.get('rnd3')),var.get('nm8').get(var.get('rnd5'))):
                            var.put('rnd5', ((var.get('Math').callprop('random')*var.get('nm8').get('length'))|Js(0.0)))
                        var.put('names', ((((var.get('nm6').get(var.get('rnd'))+var.get('nm7').get(var.get('rnd2')))+var.get('nm8').get(var.get('rnd5')))+var.get('nm7').get(var.get('rnd4')))+var.get('nm10').get(var.get('rnd3'))))
                    else:
                        var.put('rnd4', ((var.get('Math').callprop('random')*var.get('nm7').get('length'))|Js(0.0)))
                        var.put('rnd5', ((var.get('Math').callprop('random')*var.get('nm8').get('length'))|Js(0.0)))
                        var.put('rnd6', ((var.get('Math').callprop('random')*var.get('nm7').get('length'))|Js(0.0)))
                        var.put('rnd7', ((var.get('Math').callprop('random')*var.get('nm9').get('length'))|Js(0.0)))
                        while PyJsStrictEq(var.get('nm10').get(var.get('rnd3')),var.get('nm8').get(var.get('rnd5'))):
                            var.put('rnd5', ((var.get('Math').callprop('random')*var.get('nm8').get('length'))|Js(0.0)))
                        while PyJsStrictEq(var.get('nm9').get(var.get('rnd7')),var.get('nm8').get(var.get('rnd5'))):
                            var.put('rnd7', ((var.get('Math').callprop('random')*var.get('nm9').get('length'))|Js(0.0)))
                        if (var.get('i')<Js(8.0)):
                            var.put('names', ((((((var.get('nm6').get(var.get('rnd7'))+var.get('nm7').get(var.get('rnd2')))+var.get('nm9').get(var.get('rnd7')))+var.get('nm7').get(var.get('rnd4')))+var.get('nm8').get(var.get('rnd5')))+var.get('nm7').get(var.get('rnd6')))+var.get('nm10').get(var.get('rnd3'))))
                        else:
                            var.put('names', ((((((var.get('nm6').get(var.get('rnd7'))+var.get('nm7').get(var.get('rnd4')))+var.get('nm8').get(var.get('rnd5')))+var.get('nm7').get(var.get('rnd2')))+var.get('nm9').get(var.get('rnd7')))+var.get('nm7').get(var.get('rnd6')))+var.get('nm10').get(var.get('rnd3'))))
            else:
                if PyJsStrictEq(var.get('tp'),Js(2.0)):
                    var.put('rnd', ((var.get('Math').callprop('random')*var.get('nm11').get('length'))|Js(0.0)))
                    var.put('rnd2', ((var.get('Math').callprop('random')*var.get('nm12').get('length'))|Js(0.0)))
                    var.put('rnd3', ((var.get('Math').callprop('random')*var.get('nm15').get('length'))|Js(0.0)))
                    if (var.get('i')<Js(3.0)):
                        while (var.get('rnd')<Js(4.0)):
                            var.put('rnd', ((var.get('Math').callprop('random')*var.get('nm11').get('length'))|Js(0.0)))
                        while (var.get('rnd3')<Js(4.0)):
                            var.put('rnd3', ((var.get('Math').callprop('random')*var.get('nm15').get('length'))|Js(0.0)))
                        var.put('names', ((var.get('nm11').get(var.get('rnd'))+var.get('nm12').get(var.get('rnd2')))+var.get('nm15').get(var.get('rnd3'))))
                    else:
                        if (var.get('i')<Js(6.0)):
                            var.put('rnd4', ((var.get('Math').callprop('random')*var.get('nm12').get('length'))|Js(0.0)))
                            var.put('rnd5', ((var.get('Math').callprop('random')*var.get('nm13').get('length'))|Js(0.0)))
                            while PyJsStrictEq(var.get('nm12').get(var.get('rnd3')),var.get('nm13').get(var.get('rnd5'))):
                                var.put('rnd5', ((var.get('Math').callprop('random')*var.get('nm13').get('length'))|Js(0.0)))
                            var.put('names', ((((var.get('nm11').get(var.get('rnd'))+var.get('nm12').get(var.get('rnd2')))+var.get('nm13').get(var.get('rnd5')))+var.get('nm12').get(var.get('rnd4')))+var.get('nm15').get(var.get('rnd3'))))
                        else:
                            var.put('rnd4', ((var.get('Math').callprop('random')*var.get('nm12').get('length'))|Js(0.0)))
                            var.put('rnd5', ((var.get('Math').callprop('random')*var.get('nm13').get('length'))|Js(0.0)))
                            var.put('rnd6', ((var.get('Math').callprop('random')*var.get('nm12').get('length'))|Js(0.0)))
                            var.put('rnd7', ((var.get('Math').callprop('random')*var.get('nm14').get('length'))|Js(0.0)))
                            while PyJsStrictEq(var.get('nm15').get(var.get('rnd3')),var.get('nm13').get(var.get('rnd5'))):
                                var.put('rnd5', ((var.get('Math').callprop('random')*var.get('nm13').get('length'))|Js(0.0)))
                            while PyJsStrictEq(var.get('nm14').get(var.get('rnd7')),var.get('nm13').get(var.get('rnd5'))):
                                var.put('rnd7', ((var.get('Math').callprop('random')*var.get('nm14').get('length'))|Js(0.0)))
                            if (var.get('i')<Js(8.0)):
                                var.put('names', ((((((var.get('nm11').get(var.get('rnd7'))+var.get('nm12').get(var.get('rnd2')))+var.get('nm14').get(var.get('rnd7')))+var.get('nm12').get(var.get('rnd4')))+var.get('nm13').get(var.get('rnd5')))+var.get('nm12').get(var.get('rnd6')))+var.get('nm15').get(var.get('rnd3'))))
                            else:
                                var.put('names', ((((((var.get('nm11').get(var.get('rnd7'))+var.get('nm12').get(var.get('rnd4')))+var.get('nm13').get(var.get('rnd5')))+var.get('nm12').get(var.get('rnd2')))+var.get('nm14').get(var.get('rnd7')))+var.get('nm12').get(var.get('rnd6')))+var.get('nm15').get(var.get('rnd3'))))
                else:
                    var.put('rnd', ((var.get('Math').callprop('random')*var.get('nm1').get('length'))|Js(0.0)))
                    var.put('rnd2', ((var.get('Math').callprop('random')*var.get('nm2').get('length'))|Js(0.0)))
                    var.put('rnd3', ((var.get('Math').callprop('random')*var.get('nm5').get('length'))|Js(0.0)))
                    if (var.get('i')<Js(3.0)):
                        while (var.get('rnd')<Js(4.0)):
                            var.put('rnd', ((var.get('Math').callprop('random')*var.get('nm1').get('length'))|Js(0.0)))
                        while (var.get('rnd3')<Js(3.0)):
                            var.put('rnd3', ((var.get('Math').callprop('random')*var.get('nm5').get('length'))|Js(0.0)))
                        var.put('names', ((var.get('nm1').get(var.get('rnd'))+var.get('nm2').get(var.get('rnd2')))+var.get('nm5').get(var.get('rnd3'))))
                    else:
                        if (var.get('i')<Js(6.0)):
                            var.put('rnd4', ((var.get('Math').callprop('random')*var.get('nm2').get('length'))|Js(0.0)))
                            var.put('rnd5', ((var.get('Math').callprop('random')*var.get('nm3').get('length'))|Js(0.0)))
                            while PyJsStrictEq(var.get('nm2').get(var.get('rnd3')),var.get('nm3').get(var.get('rnd5'))):
                                var.put('rnd5', ((var.get('Math').callprop('random')*var.get('nm3').get('length'))|Js(0.0)))
                            var.put('names', ((((var.get('nm1').get(var.get('rnd'))+var.get('nm2').get(var.get('rnd2')))+var.get('nm3').get(var.get('rnd5')))+var.get('nm2').get(var.get('rnd4')))+var.get('nm5').get(var.get('rnd3'))))
                        else:
                            var.put('rnd4', ((var.get('Math').callprop('random')*var.get('nm2').get('length'))|Js(0.0)))
                            var.put('rnd5', ((var.get('Math').callprop('random')*var.get('nm3').get('length'))|Js(0.0)))
                            var.put('rnd6', ((var.get('Math').callprop('random')*var.get('nm2').get('length'))|Js(0.0)))
                            var.put('rnd7', ((var.get('Math').callprop('random')*var.get('nm4').get('length'))|Js(0.0)))
                            while PyJsStrictEq(var.get('nm5').get(var.get('rnd3')),var.get('nm3').get(var.get('rnd5'))):
                                var.put('rnd5', ((var.get('Math').callprop('random')*var.get('nm3').get('length'))|Js(0.0)))
                            while PyJsStrictEq(var.get('nm4').get(var.get('rnd7')),var.get('nm3').get(var.get('rnd5'))):
                                var.put('rnd7', ((var.get('Math').callprop('random')*var.get('nm4').get('length'))|Js(0.0)))
                            if (var.get('i')<Js(8.0)):
                                var.put('names', ((((((var.get('nm1').get(var.get('rnd7'))+var.get('nm2').get(var.get('rnd2')))+var.get('nm4').get(var.get('rnd7')))+var.get('nm2').get(var.get('rnd4')))+var.get('nm3').get(var.get('rnd5')))+var.get('nm2').get(var.get('rnd6')))+var.get('nm5').get(var.get('rnd3'))))
                            else:
                                var.put('names', ((((((var.get('nm1').get(var.get('rnd7'))+var.get('nm2').get(var.get('rnd4')))+var.get('nm3').get(var.get('rnd5')))+var.get('nm2').get(var.get('rnd2')))+var.get('nm4').get(var.get('rnd7')))+var.get('nm2').get(var.get('rnd6')))+var.get('nm5').get(var.get('rnd3'))))
            var.get('result').callprop('push', var.get('names'))
        finally:
                (var.put('i',Js(var.get('i').to_number())+Js(1))-Js(1))
    pass
    return var.get('result')
PyJsHoisted_nameGen_.func_name = 'nameGen'
var.put('nameGen', PyJsHoisted_nameGen_)
var.put('nm1', Js([Js(''), Js(''), Js(''), Js(''), Js('bh'), Js('br'), Js('ch'), Js('dr'), Js('g'), Js('gr'), Js('j'), Js('k'), Js('kr'), Js('q'), Js('qr'), Js('r'), Js('rh'), Js('th'), Js('thr'), Js('tr'), Js('v'), Js('y'), Js('z')]))
var.put('nm2', Js([Js('a'), Js('i'), Js('o'), Js('u'), Js('a'), Js('i'), Js('o'), Js('u'), Js('a'), Js('i'), Js('o'), Js('u'), Js('a'), Js('i'), Js('o'), Js('u'), Js('a'), Js('i'), Js('o'), Js('u'), Js('a'), Js('i'), Js('o'), Js('u'), Js('a'), Js('i'), Js('o'), Js('u'), Js('a'), Js('i'), Js('o'), Js('u'), Js('a'), Js('i'), Js('o'), Js('u'), Js('a'), Js('i'), Js('o'), Js('u'), Js('ao'), Js('au'), Js('ou'), Js('iu'), Js('oa'), Js('ua'), Js('uu')]))
var.put('nm3', Js([Js('g'), Js('gr'), Js('h'), Js('j'), Js('k'), Js('kz'), Js('kx'), Js('kn'), Js('lt'), Js('ll'), Js('m'), Js('mz'), Js('nt'), Js('nj'), Js('r'), Js('rq'), Js('rs'), Js('rz'), Js('rn'), Js('rl'), Js('sh'), Js('sz'), Js('shr'), Js('szr'), Js('t'), Js('tt'), Js('th'), Js('tr'), Js('thr'), Js('tz'), Js('tsz'), Js('tl'), Js('x'), Js('xl'), Js('z'), Js('zz')]))
var.put('nm4', Js([Js('c'), Js('cc'), Js('ch'), Js('g'), Js('j'), Js('k'), Js('kk'), Js('q'), Js('x'), Js('xx'), Js('z')]))
var.put('nm5', Js([Js(''), Js(''), Js(''), Js('k'), Js('sz'), Js('sk'), Js('shk'), Js('t'), Js('x'), Js('xl'), Js('z'), Js('zk')]))
var.put('nm6', Js([Js(''), Js(''), Js(''), Js(''), Js(''), Js(''), Js('b'), Js('cr'), Js('ch'), Js('d'), Js('dh'), Js('dr'), Js('h'), Js('j'), Js('k'), Js('kh'), Js('kr'), Js('q'), Js('r'), Js('rh'), Js('s'), Js('sh'), Js('sr'), Js('shr'), Js('th'), Js('thr'), Js('y'), Js('z')]))
var.put('nm7', Js([Js('a'), Js('e'), Js('i'), Js('o'), Js('u'), Js('a'), Js('e'), Js('i'), Js('o'), Js('u'), Js('a'), Js('e'), Js('i'), Js('o'), Js('u'), Js('a'), Js('e'), Js('i'), Js('o'), Js('u'), Js('a'), Js('e'), Js('i'), Js('o'), Js('u'), Js('a'), Js('e'), Js('i'), Js('o'), Js('u'), Js('a'), Js('e'), Js('ae'), Js('ie'), Js('ia'), Js('ei')]))
var.put('nm8', Js([Js('c'), Js('cc'), Js('ch'), Js('d'), Js('dr'), Js('g'), Js('gn'), Js('gz'), Js('gy'), Js('k'), Js('kk'), Js('kr'), Js('kz'), Js('kl'), Js('kn'), Js('lq'), Js('lz'), Js('n'), Js('nq'), Js('nr'), Js('nz'), Js('r'), Js('rs'), Js('rq'), Js('rt'), Js('rz'), Js('rsh'), Js('rzh'), Js('t'), Js('tt'), Js('th'), Js('thr'), Js('thl'), Js('tl'), Js('z'), Js('zd')]))
var.put('nm9', Js([Js('c'), Js('j'), Js('k'), Js('q'), Js('r'), Js('s'), Js('t'), Js('x'), Js('y'), Js('z')]))
var.put('nm10', Js([Js(''), Js(''), Js(''), Js(''), Js('h'), Js('s'), Js('sk'), Js('ss'), Js('x'), Js('z')]))
var.put('nm11', Js([Js(''), Js(''), Js(''), Js(''), Js(''), Js('b'), Js('ch'), Js('d'), Js('dr'), Js('j'), Js('k'), Js('kh'), Js('kr'), Js('q'), Js('r'), Js('rh'), Js('sr'), Js('shr'), Js('th'), Js('thr'), Js('y'), Js('z')]))
var.put('nm12', Js([Js('a'), Js('e'), Js('i'), Js('o'), Js('u'), Js('a'), Js('e'), Js('i'), Js('o'), Js('u'), Js('a'), Js('e'), Js('i'), Js('o'), Js('u'), Js('a'), Js('e'), Js('i'), Js('o'), Js('u'), Js('a'), Js('e'), Js('i'), Js('o'), Js('u'), Js('a'), Js('e'), Js('i'), Js('o'), Js('u'), Js('a'), Js('e'), Js('u'), Js('u'), Js('u'), Js('ae'), Js('ia'), Js('ao'), Js('au'), Js('ou'), Js('oa')]))
var.put('nm13', Js([Js('g'), Js('gr'), Js('k'), Js('kk'), Js('kz'), Js('kl'), Js('kn'), Js('lz'), Js('r'), Js('rs'), Js('rq'), Js('rt'), Js('rz'), Js('t'), Js('tt'), Js('th'), Js('thr'), Js('tl'), Js('z'), Js('zd')]))
var.put('nm14', Js([Js('c'), Js('j'), Js('k'), Js('q'), Js('r'), Js('t'), Js('x'), Js('z')]))
var.put('nm15', Js([Js(''), Js(''), Js(''), Js(''), Js('s'), Js('sk'), Js('ss'), Js('x'), Js('xl'), Js('z')]))
pass
pass


# Add lib to the module scope
lizardfolkNames = var.to_python()